WhateverGreen/kern_igfx.cpp

```diff
@@ -198,6 +199,18 @@ void IGFX::processKernel(KernelPatcher &patcher, DeviceInfo *info) {
 		KernelPatcher::RouteRequest request("__ZN9IOService20copyExistingServicesEP12OSDictionaryjj", wrapCopyExistingServices, orgCopyExistingServices);
 		patcher.routeMultiple(KernelPatcher::KernelID, &request, 1);
 	}
+	
+	// Enable CFL backlight patch if CPU gen is CFL and model is laptop
+	cflBacklightPatch = (cpuGeneration == CPUInfo::CpuGeneration::CoffeeLake && WIOKit::getComputerModel() == WIOKit::ComputerModel::ComputerLaptop);
+	cflBacklightPatch = cflBacklightPatch || (info->videoBuiltin && info->videoBuiltin->getProperty("enable-cflbklt"));
```

Comment:
Was this tested on 10.13.6 special build? If not, perhaps it is best to add a `getKernelVersion() >= KernelVersion::Mojave` check just in case?